covsirphy/analysis/scenario.py

```diff
@@ -1033,7 +1146,7 @@ def _fit_create_data(self, oxcgrt_data, model, name):
         X_target = oxcgrt_df.loc[dates]
         return (X, y, X_target)
 
-    def fit(self, oxcgrt_data, name="Main", test_size=0.2, seed=0):
+    def fit(self, oxcgrt_data, name="Main", test_size=0.2, seed=0, delay=None):
```

Comment:
Please add explanation of `delay` argument to `_fit_create_data()` and `fit()`.

Reply:
Then, could you add "delay" key to the output dicstionary of `.fit()`? This will be useful for users to debug.